### PR TITLE
Update cta-bus-map.md

### PR DESCRIPTION
**Pull request by leoj3n**
- Hide JS editor of final example.
- Consistently capitalize list items.
- Change the "what's provided" tone for better clarity.
  - Before it sounded like steps that should be followed.

### CODE REVIEW

File: docs/can-guides/commitment/recipes/cta-bus-map/cta-bus-map.md [hunk 1]
```diff
@@ -68,7 +62,7 @@ __A Basic CanJS Setup__
   ```
 
 - CanJS uses [can-stache] to render data in a template
-  and keep it live.  Templates can be authored in `<script>` tags like:
+  and keep it live.  Templates are authored in `<script>` tags like:
```
Comment (justinbmeyer) [inline docs/can-guides/commitment/recipes/cta-bus-map/cta-bus-map.md]: they shouldn't be in a real app.

**Comment by justinbmeyer** [comment]
Thanks for suggestions.  Incorporated into another commit.

**Comment by leoj3n** [comment]
image

Think this looks better / less confusing.

File: docs/can-guides/commitment/recipes/cta-bus-map/cta-bus-map.md (hunk 1 above)
Comment (leoj3n) [reply docs/can-guides/commitment/recipes/cta-bus-map/cta-bus-map.md]: well, they ARE authored in `<script>` tags _for this example_ which is why I found this sentence confusing — perhaps you could add a short blurb to qualify that a real app would organize templates into files.

**Pull request closed without merging**